blog/exporters.md

```diff
@@ -60,9 +63,10 @@ sudo skydive agent -c /etc/skydive/skydive.yml
 Be sure to set the field `analyzers` in the `skydive.yml` to point to the analyzer.
 
 
-## Create Security Advisor (Pipeline) Configuration File
+## Create (Security Advisor) Exporter Configuration File
 
-Start from the `secadvisor.yml.default` in the secadvisor directory.
+Start from the `secadvisor.yml.default` in the `secadvisor` directory.
+Different applications can be built up by specifying different parameters in the configuration file.
```

Comment:
Same here ... I expect to have flow_exporter.yml ... and secadvisor.yml.default can be additional one ... but this doc should point to the generic one and not to secadvisor.

Reply:
Again - for now it should still point to `secadvisor.yml.default` as the blog must accurately represent the current state - in future  based on changes we make to the code base (such as moving to use `uber` which includes also the secadvisor specific phases) we can adapt this blog